src/pages/LogIn/components/LoginForm.tsx

```diff
@@ -31,25 +33,23 @@ const LoginFormComponent = () => {
       });
       history.push('/');
     } else {
-      console.log('login failed');
+      setIsValidCredentials(false);
+      setHelperText("Invalid username or password.");
     }
   }
 
   return (
     <LoginForm
-      showHelperText={showHelperText}
-      helperText="Invalid login credentials."
+      showHelperText={!!helperText}
+      helperText={helperText}
       usernameLabel="Username"
       usernameValue={usernameValue}
       onChangeUsername={(val) => setUsernameValue(val)}
-      isValidUsername={isValidUsername}
+      isValidUsername={isValidCredentials}
       passwordLabel="Password"
       passwordValue={passwordValue}
       onChangePassword={(val) => setPasswordValue(val)}
-      isValidPassword={true}
-      rememberMeLabel="Keep me logged in for 30 days."
-      isRememberMeChecked={RememberMeClick}
-      onChangeRememberMe={() => setRememberMeClick(!RememberMeClick)}
+      isValidPassword={isValidCredentials}
```

Comment:
Just an overall style question: I notice that the `Invalid username or password` message is dispalyed at the top of the log-in form when incorrect information is input for log-in. From personal experience and preference, I was expecting it to show at the bottom. I'd say that I'm more used to seeing it on the bottom, instead of top.

Something very minor, but just wanted to raise here to get some opinions.

@boris-nguyen any thoughts on this?

Reply:
@colinywu in my opinion, we should keep it as it is since its part of the Patternfly component.